server/schema/auth/signup.js

```diff
@@ -27,6 +29,12 @@ const schema = {
       maxlength: 10,
       pattern: '^[0-9]{10}$',
     },
+    department: {
+      type: 'number',
+    },
+    branch: {
+      type: 'number',
+    },
```

Comment:
min and max for department and branch should be specified for validation

Reply:
done